source/navitiacommon/navitiacommon/models.py

```diff
@@ -346,11 +346,14 @@ def get_by_name(cls, name):
     @classmethod
     def get_from_id_or_name(cls, id=None, name=None):
         if id:
-            return cls.query.get_or_404(id)
```

Comment:
What is weird is that flask-sqlalchemy abort() does not complies with falsk-restful resource (flask_restful.Resource).
Flask builds on external modules, possibly not cross-consistent as they are external and independant.
Django is better for consistency indeed !